ArduCopter/Copter.h

```diff
@@ -319,14 +319,14 @@ class Copter : public AP_HAL::HAL::Callbacks {
             uint8_t throttle_zero           : 1; // 15      // true if the throttle stick is at zero, debounced, determines if pilot intends shut-down when not using motor interlock
             uint8_t system_time_set         : 1; // 16      // true if the system time has been set from the GPS
```

Comment:
We should remove this since it's not needed anymore. And while at it, just remove the numbers from the comments instead of keeping updating them.

Reply:
+1 those diffs are annoying

Reply:
Done.